docs/developer-guide/nodejs-api.md

```diff
@@ -49,15 +49,15 @@ var codeLines = SourceCode.splitLines(code);
  */
 ```
 
-## linter
+## Linter
 
-The `linter` object does the actual evaluation of the JavaScript code. It doesn't do any filesystem operations, it simply parses and reports on the code. In particular, the `linter` object does not process configuration objects or files. You can retrieve `linter` like this:
+The `Linter` object does the actual evaluation of the JavaScript code. It doesn't do any filesystem operations, it simply parses and reports on the code. In particular, the `linter` object does not process configuration objects or files. You can retrieve `linter` like this:
 
 ```js
-var linter = require("eslint").linter;
+var Linter = require("eslint").Linter;
```

Comment:
In this example, it might be a good idea to demonstrate creating an instance of `Linter`.

```js
var Linter = require("eslint").Linter;
var linter = new Linter();
```